credential-registry/server/django-icat-api/README.rst

```diff
@@ -0,0 +1,51 @@
+======
+api_v2
+======
+
+api_v2 is a Django app which provides the back-end storage and search capabilities 
+for Indy Catalyst Credential Registry.
+
+Detailed documentation is in the "docs" directory.
+
+Quick start
+-----------
+
+1. Add "api_v2" to your INSTALLED_APPS setting like this::
+
+    INSTALLED_APPS = [
+        ...
+        'api_v2',
+    ]
+
+2. Include the api_v2 URLconf in your project urls.py like this::
+
+    path("api/v2/", include("api_v2.urls")),
+
+3. Move the api_v2 folder (remove from TheOrgBook repo) into this directory
+
+4. Run "python setup.py sdist" to create a distribution
+
+5. Include the following in your Docker build pipeline to build the OrgBook image:
+
+build-api() {
```

Comment:
I'm not too familiar with .rst but this doesn't seem to be rendering properly for me

Reply:
Fixed